### DISCUSSION TITLE
Welcome to instagrapi-rest Discussions!

### DISCUSSION THREAD

**Discussion by adw0rd** [subzeroid/aiograpi-rest#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pigwa88** [comment]
I wonder if the download of information concerns only our own account on which we have to log in or it is possible to download publicly available information, photos, stories about each selected user? stupid but important question ^^

**Reply by adw0rd** [reply]
> I wonder if the download of information concerns only our own account on which we have to log in or it is possible to download publicly available information, photos, stories about each selected user? stupid but important question ^^

I didn't understand the question, but any public content  you can be download
